run_profile

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# Environment for building and running tests with dynamically loading libraries.
+#
+# ¡Keep it MacOS/Ubuntu compatible!
+
+# Use the Java that Maven uses.
+#
+# Unfortunately, a simple `which java` will not work for some users (e.g., jenv),
+# hence this a bit complex thing.
+export JAVA_HOME="$(mvn --version | grep 'Java home' | sed 's/.*: //')"
+echo "JAVA_HOME=${JAVA_HOME}"
+
+# Find the directory containing libjvm (the relative path has changed in Java 9).
+JAVA_LIB_DIR="$(find ${JAVA_HOME} -type f -name libjvm.* | xargs -n1 dirname)"
+echo "JAVA_LIB_DIR=${JAVA_LIB_DIR}"
+
+# Modify path where an executable look for libjvm.
+export RUSTFLAGS="${RUSTFLAGS:-} -C link-args=-Wl,-rpath,${JAVA_LIB_DIR}"
```

Comment:
waaait, it means that user will need to compile App from sources on his machine which is unacceptable

Reply:
Travis already compiles it, nothing changes here.
These scripts are used to build EJB for tests, whoever does it.

Reply:
We don't have a build script 'for tests'. If I understand this correctly, the final executable (or shared library, IDK) will have a path to some libjvm that was used on whatever system we build it. It might not be a problem if there is no JDK by this path on the running system, but will be if there is and it is not the one user wants to use.